### ISSUE TITLE
CP error when assigning social image from field

### ISSUE THREAD

**Issue by mikemartin** [statamic/seo-pro#97]
After setting the section default image field, the following error is shown at 
/cp/seo-pro/section-defaults/collections/articles/edit
https://flareapp.io/share/LPdn9z5O#F71

`image: '@seo:photo'`

**Collection config:**
```
title: Articles
template: articles/show
layout: layout
mount: c6d769d3-f172-4e91-91f3-c0421ca2e5bb
taxonomies:
  - categories
revisions: false
route: '/{slug}'
date: true
sort_dir: asc
date_behavior:
  past: public
  future: private
inject:
  seo:
    image: '@seo:photo'
```

**Article blueprint**:
```
title: Article
sections:
  main:
    display: Main
    fields:
      -
        handle: description
        field:
          type: textarea
          localizable: false
          listable: hidden
          display: Description
      -
        import: content
  cta:
    display: CTA
    fields:
      -
        import: cta
        prefix: cta_
  sidebar:
    display: Sidebar
    fields:
      -
        handle: author
        field:
          type: users
          display: Author
          default: current
          localizable: true
          max_items: 1
      -
        handle: photo
        field:
          mode: grid
          container: assets
          restrict: false
          allow_uploads: true
          max_files: 1
          type: assets
          localizable: false
          listable: hidden
          display: Photo
```

**SEO Pro Asset config:**
```
<?php

return [

    'assets' => [
        'container' => 'assets',
        'open_graph_preset' => [
            'w' => 1200,
            'h' => 1200,
            'fit' => 'crop',
        ],
    ],

];
```

**Comment by mikemartin** [comment]
I seem to be getting this same error when setting the default site image as well. Here are my default settings.

```
title: '@seo:hero_title'
description: '@seo:hero_summary'
site_name: 'The Depression Project'
site_name_position: after
site_name_separator: '|'
image: home/header-people.png
priority: 0.5
change_frequency: monthly
```

And now I'm receiving this error on every page:
<img width="1308" alt="Screen Shot 2020-08-05 at 9 51 11 am" src="https://user-images.githubusercontent.com/414211/89356467-3d345380-d701-11ea-83fd-ebaf6e0929a4.png">

Error log here:
https://gist.github.com/mikemartin/7af2a4d60094a8149e9318313bb44c8e

**Comment by sg-modlab** [comment]
Just noting that I am getting this same error as in the original post after setting an asset container in config and then trying to access section defaults setting page. If better for me to create new issue. Let me know.

**Comment by jesseleite** [comment]
Fixed in 2.0.5 (will tag soon).
